### PR TITLE
Issue 362 - Changed Moments spec to fixtures

### PR DESCRIPTION
**Pull request by liirene1**
Issue 362 - Changed Moments spec to fixtures

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,6 +168,8 @@ There are three config files: `config/env/test.env`, `config/env/development.env
 Copy the sample files to create your own configuration files:
 
 `cp config/env/test.example.env config/env/test.env`
+<<<<<<< HEAD
```
Comment (julianguyen) [inline README.md]: Please remove this

**Comment by julianguyen** [comment]
Hey @liirene1, any updates on this? :)

File: Errors.md [hunk 2]
```diff
@@ -116,15 +116,15 @@ development: &default
   password: <%= ENV["PSQL_PASSWORD"] %>
```
Comment (julianguyen) [inline Errors.md]: Can you please remove this file entirely from your pull request?

File: README.md [hunk 3]
```diff
@@ -230,7 +230,6 @@ docker-compose -f docker-compose.yml -f docker-compose.test.yml run --rm app bas
 
 Assuming [Vagrant](https://www.vagrantup.com/docs/installation/) is setup, you can add the following line to your Vagrantfile. Make sure to add it exactly as below - with `:` and not quotes, or it may not work properly.
 
-
```
Comment (julianguyen) [inline README.md]: Can you also remove this file from the PR too?

**Comment by liirene1** [comment]
Hi Julia,
I did the cherry-pick. Hope this is good now. Thanks!

**Pull request merged**